### ISSUE TITLE
logits before softmax

### ISSUE THREAD

**Issue by justcho5** [jamesdolezal/slideflow#18]
Which do you consider as the logits layer when you get the raw predictions before softmax? I just copy pasted the last several tensors.

> 
> name: import/xception_2/global_max_pooling2d/Max/reduction_indices
> values: (<tf.Tensor 'import/xception_2/global_max_pooling2d/Max/reduction_indices:0' shape=(2,) dtype=int32>,)
> 
> name: import/xception_2/global_max_pooling2d/Max
> values: (<tf.Tensor 'import/xception_2/global_max_pooling2d/Max:0' shape=(?, 2048) dtype=float32>,)
> 
> name: import/dense_3/kernel
> values: (<tf.Tensor 'import/dense_3/kernel:0' shape=(2048, 500) dtype=float32>,)
> 
> name: import/dense_3/bias
> values: (<tf.Tensor 'import/dense_3/bias:0' shape=(500,) dtype=float32>,)
> 
> name: import/dense_3/MatMul/ReadVariableOp
> values: (<tf.Tensor 'import/dense_3/MatMul/ReadVariableOp:0' shape=(2048, 500) dtype=float32>,)
> 
> name: import/dense_3/MatMul
> values: (<tf.Tensor 'import/dense_3/MatMul:0' shape=(?, 500) dtype=float32>,)
> 
> name: import/dense_3/BiasAdd/ReadVariableOp
> values: (<tf.Tensor 'import/dense_3/BiasAdd/ReadVariableOp:0' shape=(500,) dtype=float32>,)
> 
> name: import/dense_3/BiasAdd
> values: (<tf.Tensor 'import/dense_3/BiasAdd:0' shape=(?, 500) dtype=float32>,)
> 
> name: import/dense_3/Relu
> values: (<tf.Tensor 'import/dense_3/Relu:0' shape=(?, 500) dtype=float32>,)
> 
> name: import/dense_1_2/kernel
> values: (<tf.Tensor 'import/dense_1_2/kernel:0' shape=(500, 2) dtype=float32>,)
> 
> name: import/dense_1_2/bias
> values: (<tf.Tensor 'import/dense_1_2/bias:0' shape=(2,) dtype=float32>,)
> 
> name: import/dense_1_2/MatMul/ReadVariableOp
> values: (<tf.Tensor 'import/dense_1_2/MatMul/ReadVariableOp:0' shape=(500, 2) 
> dtype=float32>,)
> 
> name: import/dense_1_2/MatMul
> values: (<tf.Tensor 'import/dense_1_2/MatMul:0' shape=(?, 2) dtype=float32>,)
> 
> name: import/dense_1_2/BiasAdd/ReadVariableOp
> values: (<tf.Tensor 'import/dense_1_2/BiasAdd/ReadVariableOp:0' shape=(2,) dtype=float32>,)
> 
> name: import/dense_1_2/BiasAdd
> values: (<tf.Tensor 'import/dense_1_2/BiasAdd:0' shape=(?, 2) dtype=float32>,)
> 
> name: import/dense_1_2/Softmax
> values: (<tf.Tensor 'import/dense_1_2/Softmax:0' shape=(?, 2) dtype=float32>,)

Thanks!
